onnxruntime/test/framework/tensor_test.cc

```diff
@@ -137,12 +138,10 @@ TEST(TensorTest, EmptyTensorTest) {
   ASSERT_STREQ(location.name, CPU);
   EXPECT_EQ(location.id, 0);
 
-  // arena is disabled for CPUExecutionProvider on x86 and JEMalloc
-#if (defined(__amd64__) || defined(_M_AMD64) || defined(__aarch64__) || defined(__loongarch__) || defined(_M_ARM64)) && !defined(USE_JEMALLOC) && !defined(USE_MIMALLOC) && !defined(ABSL_HAVE_ADDRESS_SANITIZER)
-  EXPECT_EQ(location.alloc_type, OrtAllocatorType::OrtArenaAllocator);
```

Comment:
Here I disabled ArenaAllocator when ABSL_HAVE_ADDRESS_SANITIZER is true because otherwise the allocator can mask memory usage problems.

Reply:
yes, `DoesCpuAllocatorSupportArenaUsage()` also checks `ABSL_HAVE_ADDRESS_SANITIZER`. I was trying to put all these checks in one place originally but missed some in the unit tests.